### PR TITLE
Continued Cleanup

### PR DESCRIPTION
**Pull request by becky-intelletive**

### CODE REVIEW

File: dash/frontend/src/app/modules/private/pages/cluster/add-template-constraint/add-template-constraint.component.ts [hunk 1]
```diff
@@ -44,42 +45,53 @@ export class AddTemplateConstraintComponent implements OnInit, AfterViewInit {
...
   }
 
   ngOnInit(): void {
-    this.gatekeeperService.getNamespacesByCluster(this.data.clusterId).subscribe(response => {
-      this.k8sNamespaces = response.data;
-    });
-    if (this.data.isEdit && this.data.constraint) {
-      this.setConstraintForm();
-    }
+    this.gatekeeperService.getNamespacesByCluster(this.data.clusterId)
```
Comment (kyle-intelletive) [inline dash/frontend/src/app/modules/private/pages/cluster/add-template-constraint/add-template-constraint.component.ts:71]: This subscription should get closed

File: dash/frontend/src/app/modules/private/pages/cluster/template-constraint-manifest/template-constraint-manifest.component.ts [hunk 2]
```diff
@@ -45,13 +50,19 @@ export class TemplateConstraintManifestComponent implements OnInit {
...
 
-  setEditorContent($event: any) {
-  }
+  setEditorContent($event: any) {}
 
   saveManifest() {
     this.dialogRef.close({manifestData: JSON.parse(this.rawTemplate)});
   }
 
+  onCopyClick() {
```
Comment (kyle-intelletive) [inline dash/frontend/src/app/modules/private/pages/cluster/template-constraint-manifest/template-constraint-manifest.component.ts:59]: It looks lie you're using the copy to clipboard component in the template, so is this code actually used?

File: dash/frontend/src/app/modules/private/pages/cluster/add-template-constraint/add-template-constraint.component.ts (hunk 1 above)
Comment (becky-intelletive) [reply dash/frontend/src/app/modules/private/pages/cluster/add-template-constraint/add-template-constraint.component.ts:71]: Added .pipe(take(1))

File: dash/frontend/src/app/modules/private/pages/cluster/template-constraint-manifest/template-constraint-manifest.component.ts (hunk 2 above)
Comment (becky-intelletive) [reply dash/frontend/src/app/modules/private/pages/cluster/template-constraint-manifest/template-constraint-manifest.component.ts]: Removed

**Review by kyle-intelletive — APPROVED** [review]

**Pull request merged**